### ISSUE TITLE
Planned Data Release: V16

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-analysis#601]
#### What data file(s) does this issue pertain to?
pbta-histologies.tsv
pbta-tcga-manifest.tsv           
pbta-tcga-snv-lancet.vep.maf.gz  
pbta-tcga-snv-mutect2.vep.maf.gz  
pbta-tcga-snv-strelka2.vep.maf.gz

#### What release are you using?

V15

#### Put your question or report your issue here.
- [ ] Updated TCGA MAFs per #568 
- [ ] Update TCGA manifest file to include BED file used per sample
- [ ] Add new TCGA BED files, padded and unpadded
- [ ] Remove old TCGA BED file `gencode.v19.basic.exome.hg38liftover.100bp_padded.bed`
- [ ] Update histologies file to include additional information per tickets #607, #610, #637 ~potentially #509 and #608 #245~ (@jaclyn-taroni edit per https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/601#issuecomment-603343136)

Placeholder for V16 release to include D3b team: @baileyckelly @chris-s-friedman @yuankunzhu @allisonheath

**Comment by jaclyn-taroni** [comment]
Because we made changes to the `pbta-fusion-putative-oncogenic.tsv` with v15, we've had to update the `fusion-summary` files that are included in a release (`fusion_summary_embryonal_foi.tsv` and `fusion_summary_ependymoma_foi.tsv`): https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/516201f91f174c495f6d7964f4a03d6c7c6fb227/analyses/fusion-summary/results

**Comment by allisonheath** [comment]
For the v16 release, @baileyckelly and @chris-s-friedman are moving the histology file generation over to a new database workflow to make the manual changes more trackable. We are working on replicating v15 first, expected to be done by the end of this week and then will start working on the new issues noted in the ticket early next. Will update if this changes.

**Comment by allisonheath** [comment]
And can we go ahead and add https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/624 as well?

**Comment by jaclyn-taroni** [comment]
@tkoganti gave me a heads up that the updated TCGA files are available pre-v16 if folks would like to take a look.

Here's the link to the s3 bucket and folder `https://s3.amazonaws.com/kf-openaccess-us-east-1-prd-pbta/data/TCGA_mar-12-2020/`

And the file list:

```
README.md
intersected_whole_exome_agilent_designed_120_AND_tcga_6k_genes.Gh38.bed
intersected_whole_exome_agilent_plus_tcga_6k_AND_tcga_6k_genes.Gh38.bed
pbta-tcga-manifest_all319.txt
pbta-tcga-snv-lancet.maf.gz
pbta-tcga-snv-mutect2.maf.gz
pbta-tcga-snv-strelka2.maf.gz
tcga_6k_genes.targetIntervals.bed
whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.Gh38.bed
whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed
whole_exome_agilent_designed_120.targetIntervals.Gh38.bed
whole_exome_agilent_designed_120.targetIntervals.bed
whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed
```
Where the README includes an explanation of what each of these files are.

**Comment by yuankunzhu** [comment]
[note-to-self]

```bash
# sync data from previous release
last_release='release-v15-20200228'
new_release='release-v16-20200320'
bucket='s3://kf-openaccess-us-east-1-prd-pbta/data'
aws s3 sync $bucket/$last_release/ $bucket/$new_release/
```
## new data
### TCGA
 get overlapped file names between v15 and `TCGA_mar-12-2020/`
```bash
aws s3 ls --recursive $bucket \
    | awk '{print $4}' \
    | egrep "$new_release|TCGA_mar-12-2020/" \
    | xargs -I {} basename {} \
    | sort | uniq -dc
```
it turns 0 output, no overlaps? check all TCGA MAF files:
```bash
aws s3 ls --recursive $bucket \
    | awk '{print $4}' \
    | egrep "$new_release|TCGA_mar-12-2020/" \
    | grep -i tcga | grep -i maf
```
returns below
```
data/TCGA_mar-12-2020/pbta-tcga-snv-lancet.maf.gz
data/TCGA_mar-12-2020/pbta-tcga-snv-mutect2.maf.gz
data/TCGA_mar-12-2020/pbta-tcga-snv-strelka2.maf.gz
data/release-v16-20200320/pbta-tcga-snv-lancet.vep.maf.gz
data/release-v16-20200320/pbta-tcga-snv-mutect2.vep.maf.gz
data/release-v16-20200320/pbta-tcga-snv-strelka2.vep.maf.gz
```
MAF files named differently in `TCGA_mar-12-2020/` folder and release folder, rename/overwrite TCGA_mar-12-2020/ MAF with vep extension to keep consistence
```
for caller in 'lancet' 'mutect2' 'strelka2'
do
    aws s3 cp $bucket/TCGA_mar-12-2020/pbta-tcga-snv-$caller.maf.gz $bucket/$new_release/pbta-tcga-snv-$caller.vep.maf.gz
done
```

**Comment by yuankunzhu** [comment]
@tkoganti for the new TCGA BED files you put on `TCGA_mar-12-2020/`
I didn't see the Gh38 version for `tcga_6k_genes.targetIntervals.bed` and the original/un-liftover'd for `whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed`?

**Comment by yuankunzhu** [comment]
@tkoganti I think we need to also add the BED information to the manifest.

**Comment by tkoganti** [comment]
> @tkoganti for the new TCGA BED files you put on `TCGA_mar-12-2020/`
> I didn't see the Gh38 version for `tcga_6k_genes.targetIntervals.bed` and the original/un-liftover'd for `whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed`?

Just uploaded `data/TCGA_mar-12-2020/tcga_6k_genes.targetIntervals.padded.Gh38.bed` 

This file from March 12 `data/TCGA_mar-12-2020/whole_exome_agilent_plus_tcga_6k.targetIntervals.bed` - Is this what you are asking for?

**Comment by yuankunzhu** [comment]
> This file from March 12 is in the. s3 bucket - `data/TCGA_mar-12-2020/whole_exome_agilent_plus_tcga_6k.targetIntervals.bed` - Is this what you are asking for?

```
$ aws s3 ls $bucket/TCGA_mar-12-2020/ | grep whole_exome_agilent_plus_tcga_6k.targetIntervals.bed          <aws:saml>
2020-03-20 13:22:34    7684576 whole_exome_agilent_plus_tcga_6k.targetIntervals.bed
```
it looks like it's just uploaded?

**Comment by tkoganti** [comment]
> > This file from March 12 is in the. s3 bucket - `data/TCGA_mar-12-2020/whole_exome_agilent_plus_tcga_6k.targetIntervals.bed` - Is this what you are asking for?
> 
> ```
> $ aws s3 ls $bucket/TCGA_mar-12-2020/ | grep whole_exome_agilent_plus_tcga_6k.targetIntervals.bed          <aws:saml>
> 2020-03-20 13:22:34    7684576 whole_exome_agilent_plus_tcga_6k.targetIntervals.bed
> ```
> 
> it looks like it's just uploaded?
> 
<img width="745" alt="Screen Shot 2020-03-20 at 1 33 56 PM" src="https://user-images.githubusercontent.com/56036338/77190411-929ff900-6aaf-11ea-964e-24e084f55a3f.png">

Hmm It said March 12 when I first looked and I uploaded again. Not sure if it was overwritten. 

This is the capture kit info used for the new samples. But might be easier to generate again with all 319 samples. Let me know if you want me to create that - 
[tcga-capture_kit-info_NEW.txt](https://github.com/AlexsLemonade/OpenPBTA-analysis/files/4361093/tcga-capture_kit-info_NEW.txt)

**Comment by kgaonkar6** [comment]
Adding files updated from fusion-filtering analysis https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering/results

Changes :
A QC step to identify and remove fusions in multi (more than 4) histologies is added to the fusion-filtering analysis:
Update files:
- [ ] pbta-fusion-putative-oncogenic.tsv  
- [ ] pbta-fusion-recurrently-fused-genes-byhistology.tsv
- [ ] pbta-fusion-recurrently-fused-genes-bysample.tsv

**Comment by baileyckelly** [comment]
Hi there -
From reviewing tickets, can we confirm that the following tickets are likely not going to be ready for v16 for the pbta-histologies file?
#608 - blocked
#509 - Target for v17 
#245 - Target for v17

We will be adding in the following ticket(s), though for the histologies file: 
#637 - Data refresh for pbta-histologies file

**Comment by jaclyn-taroni** [comment]
> Hi there -
> From reviewing tickets, can we confirm that the following tickets are likely not going to be ready for v16 for the pbta-histologies file?
> #608 - blocked
> #509 - Target for v17
> #245 - Target for v17
> 
> We will be adding in the following ticket(s), though for the histologies file:
> #637 - Data refresh for pbta-histologies file

Yes, sounds good. I will edit the original post to match this.

**Comment by yuankunzhu** [comment]
D3b data team is still working on clinical data updating, we are suggesting move all changes related `pbta-histologies.tsv` to v17(#656), so for this release, it will have changes for:
Updated TCGA MAFs per #568
Update TCGA manifest file to include BED file used per sample
Add new TCGA BED files, padded and unpadded
Remove old TCGA BED file gencode.v19.basic.exome.hg38liftover.100bp_padded.bed
- Updated TCGA MAFs per #568
- Update TCGA manifest file to include BED file used per sample
- Add new TCGA BED files, padded and unpadded
- Remove old TCGA BED file `gencode.v19.basic.exome.hg38liftover.100bp_padded.bed`
- new fusions files from #621:
  - `pbta-fusion-putative-oncogenic.tsv`
  - `pbta-fusion-recurrently-fused-genes-byhistology.tsv`
  - `pbta-fusion-recurrently-fused-genes-bysample.tsv`

cc @jaclyn-taroni and @baileyckelly

**Comment by jaclyn-taroni** [comment]
Closed via #657
